ui/app/AppLayouts/Wallet/views/LeftTabView.qml

```diff
@@ -104,65 +128,124 @@ Rectangle {
         }
 
         StatusListView {
+            id: walletAccountsListView
+
             objectName: "walletAccountsListView"
             spacing: Style.current.smallPadding
             Layout.fillWidth: true
             Layout.fillHeight: true
             Layout.topMargin: Style.current.halfPadding
+            Layout.leftMargin: -Style.current.padding
+            Layout.rightMargin: -Style.current.padding
 
             // ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
 
+            readonly property Item firstItem: count > 0 ? itemAtIndex(0) : null
+
             delegate: StatusListItem {
                 objectName: "walletAccountItem"
-                width: ListView.view.width
-                highlighted: RootStore.currentAccount.name === model.name
+                width: ListView.view.width - Style.current.padding * 2
+                highlighted: !ListView.view.footerItem.button.highlighted &&
+                             RootStore.currentAccount.name === model.name
+                anchors.horizontalCenter: parent.horizontalCenter
                 title: model.name
                 subTitle: LocaleUtils.currencyAmountToLocaleString(model.currencyBalance)
                 asset.emoji: !!model.emoji ? model.emoji: ""
                 asset.color: model.color
                 asset.name: !model.emoji ? "filled-account": ""
+                asset.width: 40
+                asset.height: 40
                 asset.letterSize: 14
                 asset.isLetterIdenticon: !!model.emoji ? true : false
                 asset.bgColor: Theme.palette.primaryColor3
                 statusListItemTitle.font.weight: Font.Medium
                 color: sensor.containsMouse || highlighted ? Theme.palette.baseColor3 : "transparent"
                 onClicked: {
                     changeSelectedAccount(index)
-                    showSavedAddresses(false)
+                    showSavedAddresses = false
                 }
+                components: [
+                    StatusIcon {
+                        icon: {
+                            if (model.walletType == "watch")
+                                return "show"
+                            else if (model.walletType == "key")
+                                return "keycard"
+
+                            return ""
+                        }
+                        color: Theme.palette.directColor1
+                        width: 15
+                        height: 15
+                    }
+                ]
             }
 
-            footer: Item {
-                width: ListView.view.width
-                height: addAccountBtn.height + Style.current.xlPadding
+            readonly property bool footerOverlayed: contentHeight > availableHeight
+
+            footerPositioning: footerOverlayed ? ListView.OverlayFooter : ListView.InlineFooter
+            footer: Rectangle {
```

Comment:
could you please use `Control` setting the background and contentItem properties instead?

Reply:
yes!